pkg/k8s/k8s.go

```diff
@@ -22,6 +22,8 @@ const (
 	StatefulSet           = "statefulset"
 )
 
+var SortedRes = []string{Authority, DaemonSet, Deployment, Namespace, Pod, ReplicationController, ReplicaSet, Service, ServiceProfile, StatefulSet}
```

Comment:
How would you feel about calling this `AllResources`, and adding a comment that the contents should be sorted? I think that would make it clearer that this must be updated when we add new resources in the future. I'd also split this only multiple lines:

```
// AllResources is a sorted list of all resources defined as constants above.
var SortedRes = []string{
	Authority,
	DaemonSet,
	Deployment,
	...
```

Reply:
Yep, that'd be more explicit.